coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
```

Comment:
But @Techievena ... you are right that we should move this discussion to another place to get this PR merged. Because it's so fundamental and necessary for all further aspects PRs :) And existing ones like @Asnelchristian's https://github.com/coala/coala-bears/pull/1391